library/network_connections.py

```diff
@@ -1062,12 +1068,15 @@ def connection_create(self, connections, idx, connection_current=None):
             for option in ip["dns_options"]:
                 s_ip4.add_dns_option(option)
```

Comment:
I think this is missing a check on `is_ipv4_available`.. right?

Reply:
For dns option, the following are valid configuration:
``` 
ipv6.method:                            ignore
ipv6.dns:                               --
ipv6.dns-search:                        --
ipv6.dns-options:                       debug
```

Reply:
but when ipv4.method is disabled, then the ipv4.dns-options can not be configured.

Reply:
I think they can be configured.. maybe not with nmcli because of a bug on in ´make_ip4_config()´ in NM ifcfg plugin.. But when using libnm I think it is possible which if a problem..  Am I missing something?